### ISSUE TITLE
Research Images

### ISSUE THREAD

**Issue by jackcarlisle** [dwyl/feedback#6]
<img width="1177" alt="screen shot 2017-03-31 at 09 19 13" src="https://cloud.githubusercontent.com/assets/12450298/24554452/38df2670-15f3-11e7-9cdd-8757072183e0.png">

**Comment by jackcarlisle** [comment]
![screen shot 2017-03-31 at 09 26 09](https://cloud.githubusercontent.com/assets/12450298/24554725/3864b056-15f4-11e7-9e09-748b4da8e5bc.png)

**Comment by jackcarlisle** [comment]
<img width="1668" alt="screen shot 2017-03-31 at 09 27 34" src="https://cloud.githubusercontent.com/assets/12450298/24554757/5272e8fa-15f4-11e7-8c6e-33031ffc30c5.png">

**Comment by jackcarlisle** [comment]
![screen shot 2017-03-31 at 09 35 28](https://cloud.githubusercontent.com/assets/12450298/24555098/81b8b8a0-15f5-11e7-9247-7463a41db3b9.png)

**Comment by jackcarlisle** [comment]
![screen shot 2017-03-31 at 09 44 25](https://cloud.githubusercontent.com/assets/12450298/24555419/b3c02ff8-15f6-11e7-9bf4-080b13546c41.png)

**Comment by jackcarlisle** [comment]
![screen shot 2017-03-31 at 10 10 43](https://cloud.githubusercontent.com/assets/12450298/24556562/6d0058d2-15fa-11e7-9102-7337c47af05f.png)

**Comment by jackcarlisle** [comment]
![screen shot 2017-03-31 at 10 12 35](https://cloud.githubusercontent.com/assets/12450298/24556654/b82b2490-15fa-11e7-819f-f682e18894ce.png)

**Comment by jackcarlisle** [comment]
![screen shot 2017-03-31 at 10 17 15](https://cloud.githubusercontent.com/assets/12450298/24556853/61899f8a-15fb-11e7-9c40-f38d170fcd82.png)

**Comment by jackcarlisle** [comment]
<img width="494" alt="screen shot 2017-03-31 at 10 20 56" src="https://cloud.githubusercontent.com/assets/12450298/24556996/c7a9461c-15fb-11e7-846b-1b2e929ec530.png">

**Comment by jackcarlisle** [comment]
![screen shot 2017-03-31 at 10 20 56](https://cloud.githubusercontent.com/assets/12450298/24557030/eb3f7ff6-15fb-11e7-815b-3f110b547f35.png)

**Comment by nelsonic** [comment]
@jackcarlisle is the conclusion of your initial research that there is or is _not_ a platform we can _already_ use?

**Comment by jackcarlisle** [comment]
@nelsonic I think [Suggestion Ox](https://www.suggestionox.com/) and [15FIVE](https://www.15five.com/) could be potential candidates for platforms we could already use. However they do require a subscription and set limits on the amount of feedback you can request based on the package you go for. Here are the links to the plans and pricing:

- Suggestion Ox https://www.suggestionox.com/pricing
- 15FIVE https://www.15five.com/pricing

Both are closed-source. The only open source application that was vaguely similar was PHPBack http://www.phpback.org/ but it's for product feedback. I'd say there's room in the market for an open-source anonymous feedback system. 

I've just double checked and [15FIVE](https://www.15five.com/) doesn't support anonymous feedback so I guess that just leaves [Suggestion Ox](https://www.suggestionox.com/).

What are your thoughts?

**Comment by nelsonic** [comment]
Suggestion Ox:
![image](https://cloud.githubusercontent.com/assets/194400/24633315/d6b33eda-18bf-11e7-852b-5fd7f4a38db7.png)

And 15Five 
![image](https://cloud.githubusercontent.com/assets/194400/24633341/f4544b78-18bf-11e7-96c6-40c1a81d1c2d.png)

Are _useful_ in that they are "_market validation_" but _neither_ of them is Open Source or has an API which means our data would be "_locked up_" and we have to _keep_ paying for ever unless we manually write a scraper to extract the data. We don't use services that have vendor lock-in and nobody `else` should be forced to either.

**Comment by jackcarlisle** [comment]
@nelsonic yeah I couldn't find any that fit the Open Source/API description. Seeing as we don't want to get locked in to a subscription we can conclude from my research that there isn't one that we can already use.

**Comment by nelsonic** [comment]
@jackcarlisle can you please capture this in a "Conclusion" section in [`research.md`](https://github.com/dwyl/feedback/blob/master/research.md) ?
